sdk/storage/azure_storage_blob/src/clients/blob_container_client.rs

```diff
@@ -249,4 +253,32 @@ impl BlobContainerClient {
     ) -> Result<Response<BlobContainerClientGetAccountInfoResult, NoFormat>> {
         self.client.get_account_info(options).await
     }
+
+    /// Sets the permissions for the specified container. The permissions indicate whether blobs in a
+    /// container may be accessed publicly.
+    ///
+    /// # Arguments
+    ///
+    /// * `container_acl` - The access control list for the container.
+    /// * `options` - Optional configuration for the request.
+    pub async fn set_access_policy(
+        &self,
+        container_acl: RequestContent<Vec<SignedIdentifier>, XmlFormat>,
```

Comment:
Will probably end up changing this parameter to `Vec<SignedIdentifier>` and we will call the `into()` internally on behalf of the cx before handing to generated layer.